### ISSUE TITLE
[A11Y]  Headings

### ISSUE THREAD

**Issue by erikkroes** [OpenConext/OpenConext-myconext#1064]
## Finding

Headings are used with varying levels, and varying adequacy.

## Location

Various pages.

## Problem

Headings are often used as a navigational aid. 
1. They should be used with proper heading levels, to create a sort of tree-structure.
2. They should have useful content so they are understandable, and properly describe the text they are "responsible for"
3. They should not be missapropriated for things that are not headings, or vice versa, things that should be headings should not be made to look like headings while not using the proper HTML

As a bonus, proper headingstructure also affects findability and SEO.

## Solution

Use a tool like Headingsmap to inspect the heading structure of pages. Make sure every page has 1 proper `<h1>` (preferably aligned with the title), no levels are skipped and headings are aligned with their content (headings cover all content, and all content is covered by the headings).

### Voorbeeld ~ @Tyskai 
Zie mijn eerste comment. Oa:
<img width="1592" height="1064" alt="Image" src="https://github.com/user-attachments/assets/186ff7bc-4007-401d-b399-d640e3c61794" />

Mocht je wel tegen gekke header shifts aanlopen, let us know in the comments

## Resources

- https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html
- https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels.html
- https://www.w3.org/WAI/WCAG22/Understanding/section-headings.html
- https://www.erikkroes.nl/blog/semantics-what-does-it-mean/
- Screen Reader User Survey #10: https://webaim.org/projects/screenreadersurvey10/#finding
- Headingsmap browser extension: https://github.com/dzc34/headingsMap
